app/src/main/java/de/danoeh/antennapod/menuhandler/FeedMenuHandler.java

```diff
@@ -75,4 +86,36 @@ protected void updateSort(@NonNull SortOrder sortOrder) {
         sortDialog.openDialog();
     }
 
+    public static boolean onMenuItemClicked(@NonNull Fragment fragment, int menuItemId,
```

Comment:
`SubscriptionFragment.java` should use this method. Currently there is code duplication.